core/src/main/scala/org/apache/spark/sql/execution/CoprocessorRDD.scala

```diff
@@ -313,30 +295,32 @@ case class RegionTaskExec(child: SparkPlan,
               logger.info("Single batch handles size:" + handleList.size())
               // After `splitAndSortHandlesByRegion`, ranges in the task are arranged in order
               // TODO: Maybe we can optimize splitAndSortHandlesByRegion if we are sure the handles are in same region?
-              val task = proceedTasksOrThrow(
+              val indexTask =
```

Comment:
if not a single task after split, please name it indexTasks.

Reply:
done